lib/YaST/DNSServer/DNSServerController.pm

```diff
@@ -0,0 +1,91 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The class introduces actions for DNS Server Settings Page.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YaST::DNSServer::DNSServerController;
+use strict;
+use warnings;
+use YaST::DNSServer::StartUpPage;
+use YuiRestClient;
+
+sub new {
+    my ($class, $args) = @_;
+    my $self = bless {}, $class;
+    return $self->init($args);
+}
+
+sub init {
+    my ($self, $args) = @_;
+    $self->{StartUpPage} = YaST::DNSServer::StartUpPage->new({app => YuiRestClient::get_app()});
+    $self->{WeakPasswordPopup} = Installation::Popups::YesNoPopup->new({app => YuiRestClient::get_app()});
```

Comment:
if this works without being declared on top, I guess we can also remove line 12, right?

Reply:
The test failed if I removed line 12, https://openqa.suse.de/tests/10069988#step/yast2_dns_server_service_inactive_enabled/2 . I think it should be the StartUpPage inherit 'YaST::Navigation::NavigationBase', if not loaded in controller then can't find the init method.

Reply:
but it works without importing `Installation::Popups::YesNoPopup` interesting...

Reply:
Yes, I will have a try to remove the inherit of YaST::Navigation::NavigationBase in StartUpPage later to see whether I can remove the line 12, I do want to know the root cause. :)

Reply:
It still fail even removed the inherit of YaST::Navigation::NavigationBase in StartUpPage, no idea why this happened.